### PR TITLE
feat: adding fiveg_rfsim interface

### PR DESCRIPTION
**Pull request by gatici**
A new charm relation interface between the OAI NR UE and the OAI DU according to Spec: https://docs.google.com/document/d/1Tv3MHXcultURY5sI0zbOiNDo65li7p2GpXXdlnPebQE/edit

### CODE REVIEW

File: interfaces/fiveg_rfsim/v0/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+As with all Juju relations, the `fiveg_rfsim` interface consists of two parties: a Provider and a Requirer.
+
+## Behavior
+
+Both the Requirer and the Provider need to adhere to criteria to be considered compatible with the interface.
+
+### Provider
+
+- Is expected to provide the DU's `rfsim` service address including pod ip and service port.
```
Comment (ghislainbourgeois) [inline interfaces/fiveg_rfsim/v0/README.md:26]:
```suggestion
- Is expected to provide the DU's `rfsim` service ip.
```

File: interfaces/fiveg_rfsim/v0/schema.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+"""
+
+from pydantic import BaseModel, Field
+
+from interface_tester.schema_base import DataBagSchema
+
+
+class FivegRFSIMProviderAppData(BaseModel):
+    rfsim_address: str = Field(
+        description="RF simulator service address which is equal to DU pod ip",
```
Comment (ghislainbourgeois) [inline interfaces/fiveg_rfsim/v0/schema.py:23]:
```suggestion
        description="RF simulator service IP",
```

**Review by ghislainbourgeois — CHANGES_REQUESTED** [review]

File: interfaces/fiveg_rfsim/v0/README.md (hunk 1 above)
Comment (gatici) [reply interfaces/fiveg_rfsim/v0/README.md]: Thank you. Done.

File: interfaces/fiveg_rfsim/v0/schema.py (hunk 2 above)
Comment (gatici) [reply interfaces/fiveg_rfsim/v0/schema.py]: Done

**Review by ghislainbourgeois — APPROVED** [review]

**Comment by gatici** [comment]
Hello @tonyandrewmeyer ,
I appreciate if you can review it.

**Review by tonyandrewmeyer — APPROVED** [review]
LGTM, thanks!

**Comment by gatici** [comment]
@tonyandrewmeyer  Many thanks for the review. Can you merge it ? I do not have rights to do it.

**Pull request merged**